src/main/java/seedu/address/logic/commands/SellCommand.java

```diff
@@ -22,7 +22,7 @@ public class SellCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a transaction and links to the company. "
             + "Parameters: "
-            + "Index "
+            + "Index (must be a positive integer) "
```

Comment:
All seems good! Perhaps adding non-zero positive integer would be more informative.